src/main.js

```diff
@@ -1 +1,12 @@
-console.log("Car");
+class Car {
+  static FORWARD_STEP = 1;
+
+  constructor(name, position = 0) {
+    this.name = name;
+    this.position = position;
+  }
+
+  forward(position = FORWARD_STEP) {
```

Comment:
```suggestion
  forward(position = Car.FORWARD_STEP) {
```

아마 이렇게 해야 정상동작 할 것 같아요..!

Reply:
<img width="559" height="522" alt="image" src="https://github.com/user-attachments/assets/b48ade2d-eeec-49e8-9271-a77bd7c9b6f3" />

지금은 오류가 발생한답니다 ㅎㅎ

Reply:
@JunilHwang 
node로 js를 실행해보니 오류가 발생하는군요. 😭 
오류 수정하였습니다. 🙇 
b49a062726587c66be5527d6403a30b0686ad56c